torch/testing/_asserts.py

```diff
@@ -723,6 +822,12 @@ def assert_close(
     If :attr:`actual` and :attr:`expected` are complex-valued, they are considered close if both their real and
     imaginary components are considered close according to the definition above.
 
+    If :attr:`actual` and :attr:`expected` are sparse (either having COO or CSR layout), their strided members are
+    checked individually. Indices, namely ``indices`` for COO or ``crow_indices``  and ``col_indices`` for CSR layout,
+    are always checked for equality whereas the values are checked for closeness according to the definition above.
+    Sparse COO tensors are only considered close if both are either coalesced or non-coalesced (if
```

Comment:
Nit: we use "uncoalesced" elsewhere in the code/docs/design document rather than "non-coalesced", although, the latter sounds good to me as well. Either way, it would be good to be consistent with the used terminology.